src/main/java/org/onebusaway/io/client/util/RegionUtils.java

```diff
@@ -27,10 +27,7 @@
 import java.net.URL;
 import java.text.DecimalFormat;
 import java.text.NumberFormat;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.HashMap;
-import java.util.List;
+import java.util.*;
```

Comment:
@philipmw Can you change this to just import `Collection`, and switch up your IDE settings to explicitly import used classes (vs. `.*`)?

I've been guilty of not using the [OBA Eclipse template](https://github.com/OneBusAway/onebusaway/wiki/Code-Style) too, and I've noticed some similar commit noise on the Alexa project.  For this and Alexa project I believe I've been using the IntelliJ default.  We should settle on using the same template.